src/Native/Runtime/EHHelpers.cpp

```diff
@@ -223,6 +223,8 @@ COOP_PINVOKE_HELPER(void, RhpCopyContextFromExInfo,
     pContext->Sp  = pPalContext->SP;
     pContext->Lr  = pPalContext->LR;
     pContext->Pc  = pPalContext->IP;
+#elif defined(_ARM64_)
+    throw "FIXME:ARM64";
```

Comment:
One of the design points of the low level runtime is that no C++ EH and no Win32 SEH is allowed.  Managed EH isn't even allowed to dispatch across the C++ code frames except in very specific circumstances that are all handled specially.  We compile (on Windows, for UWP apps) with C++ EH disabled, so adding C++ throws is not really in keeping with the style of the codebase.  You should use `PORTABILITY_ASSERT("@TODO: FIXME:ARM64")` in all these places that you've added throws. This kind of assert is supposed to trigger in both retail and debug builds, but in reviewing the code, it looks like that no longer happens.  We can fix that separately.